clusterctl/examples/google/generate-yaml.sh

```diff
@@ -21,7 +21,9 @@ set -o pipefail
 GCLOUD_PROJECT=$(gcloud config get-value project)
 ZONE=$(gcloud config get-value compute/zone)
 ZONE="${ZONE:-us-central1-f}"
-CLUSTER_NAME=test1
+# Generate a somewhat unique cluster name, UUID is not an option as service-accounts are limited to 30 characters in length
+# and have a 19 character prefix (i.e. 'machine-controller-')
+CLUSTER_NAME=$(head -c11 < <(tr -dc 'a-zA-Z0-9' < /dev/urandom) | tr '[:upper:]' '[:lower:]')
```

Comment:
Where did you test this command? Can you verify that it works on both mac and linux?

Reply:
I tested on mac -- are you getting that on linux?

Reply:
Just tried on a GCP VM, it's working for me there as well.

Reply:
that was the output on my mac.

Reply:
Turns out the default 'tr' on OSX is trying to interpret everything as UTF8 so it doesn't like the non-UTF8 characters coming out of urandom. Prefixing 'tr' with `LC_ALL=C` will tell tr that the locale is the C standard and therefore the input is treated as simply bytes. Updating with this change.

Tested on both a GCP VM, my mac with gnutools tr, and on my mac with the default OSX tr at /usr/bin/tr

Reply:
Updated the PR with the change.